config/config.go

```diff
@@ -61,8 +60,6 @@ const (
 	DefHost = "0.0.0.0"
 	// DefStatusHost is the default status host of TiDB
 	DefStatusHost = "0.0.0.0"
-	// DefStoreLivenessTimeout is the default value for store liveness timeout.
-	DefStoreLivenessTimeout = "5s"
```

Comment:
Do we have backward compatibility issue here?

Reply:
nop